src/components/Accordion/Accordion/__stories__/accordion.stories.mdx

```diff
@@ -9,6 +9,17 @@ import { accordionSingleActivePlaySuite, accordionMultiActivePlaySuite } from ".
 <Meta
   title="Data display/Accordion"
   component={ Accordion }
+  subcomponents = {{AccordionItem}}
+  argTypes={{
+    children: { control: false },
+    defaultIndex: { control: false }
+  }}
+  parameters={{
+    controls: {
+      expanded: true,
+      sort: 'requiredFirst'
```

Comment:
awesome, wouldn't we like to apply this by default to all stories? maybe with storybook's global config?

Reply:
Yes we want. This is just an example on one of Stories. We can apply it to all stories according to:
https://storybook.js.org/docs/react/essentials/controls#show-full-documentation-for-each-property